src/upcoming-components/Accordion/Accordion.tsx

```diff
@@ -26,7 +16,7 @@ export interface Props {
  * import {Accordion} from "@chanzuckerberg/eds";
  * ```
  *
- * TODO: update this comment with a description of the component.
+ * Component is an accordian panel element with additional styling passing in
```

Comment:
For the component comments, could you try to describe what the component does? I don't think we need the "with additional styling passed in" or mentioning the props, just a basic description of the purpose of the thing. In this case, an accordion is a common component type, so I googled "accordion component" and grabbed the description from the first result https://mui.com/material-ui/react-accordion/

```suggestion
 * Component that allows the user to show and hide sections of related content on a page.
```